app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -61,30 +61,26 @@
 
           <TextView
             android:id="@+id/admin_auth_heading_textview"
+            style="@style/Heading2"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="72dp"
             android:layout_marginTop="44dp"
             android:layout_marginEnd="72dp"
-            android:fontFamily="sans-serif-medium"
             android:maxLines="2"
             android:text="@string/admin_auth_heading"
-            android:textColor="@color/oppiaPrimaryText"
-            android:textSize="20sp"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintHorizontal_bias="0.0"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
           <TextView
             android:id="@+id/admin_auth_sub_text"
+            style="@style/Subtitle1"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="12dp"
-            android:fontFamily="sans-serif"
             android:text="@string/admin_auth_sub"
-            android:textColor="@color/oppiaAdminAuthSecondaryText"
```

Comment:
Please add this color back. You can notice that color this textview is different in screenshots.

Reply:
Yes there's a slight change now i've added it